app/decorators/audio_work_show_decorator.rb

```diff
@@ -88,6 +90,22 @@ def combined_mp3_audio
     model&.oral_history_content&.combined_audio_mp3&.url(public:true)
   end
 
+  def combined_mp3_audio_download
+    base = "Combined audio for #{model.title}".
+      gsub(/([[:space:]]|[[:punct:]])+/, '_').
+      downcase
+    filename = DownloadFilenameHelper.filename_with_suffix(base, content_type: 'audio/mpeg')
```

Comment:
Hmm, you are writing literal "Combined audio for", but then immediately downcasing and replacing spaces with underscore. I understand why you need to do do the gsub/downcase for the `model.title` because that's dynamic, but for the literal, you could just start out with "combined_audio_for_", since that's what you want to end up with. 

But I'm not sure that's really what we want to end up with. We'll end up wtih download names like "combined_audio_for_title_of_work.mp3".  Is that really what we want?  

I think `title_of_work.mp3` would probably be quite suffiicent. Although we also might want to limit the number of words in case of title of work is really long. 

Let's actually just look at how the title is constructed for the analogous case of our whole-work PDF/zip downloads....

https://github.com/sciencehistory/scihist_digicoll/blob/master/app/models/on_demand_derivative.rb#L77-L84

First three words of title, plus friendlier id. (Not sure what's going on in there with `deriv_type`). Can we do something similar here, using similar code? For consistency, and because that seems like a pretty good way to construct the filename too. There's even a helper there for extracting first three words that you can re-use. 

I see you decided not to do a controller to redirect, putting the temporary S3 URL directly in the HTML. This is probably fine. I can't recall why i did the redirect situation in the case of the PDF/ZIP derivatives... one thing I can think of is, using a redirect URL makes the page containing the link long-term cacheable by an HTTP cache, for longer than the S3 URL is good for. I'm not sure if this really matters or not though.

Reply:
First 3 words of title + friendlier_id - done.
I really don't mind that much about page being cached, but I figure if it becomes a problem we can come back to this!